src/tests/mocks/service_transactions_statistics_mock.js

```diff
@@ -0,0 +1,617 @@
+const nock = require('nock')
+const moment = require('moment')
+
+const { services } = require('../../config')
+
+nock(services.ADMINUSERS_URL + '/v1/api/services')
+  .get('/list')
+  .reply(200, [
+    {
```

Comment:
We have a (very rough) idea of a fixtures folder to put fixtures for mocking out responses

(for example) https://github.com/alphagov/pay-toolbox/blob/master/src/tests/fixtures/payment.ts

Separating these out makes the stubbing logic less verbose and will make it easier to do contract tests in the future. We may also not need to mock the full expected response (instead only including what we need for each test and building it up over time!) but otherwise this looks good